orchagent/aclorch.cpp

```diff
@@ -2120,6 +2120,32 @@ void AclOrch::doAclRuleTask(Consumer &consumer)
     }
 }
 
+void AclOrch::processPendingPorts()
+{
+    for (auto itmap : m_AclTables)
+    {
+        auto table = itmap.second;
+        for (auto port_alias: table.pendingPortSet)
+        {
+            SWSS_LOG_INFO("found the port: %s in ACL table: %s pending port list, bind it to ACL table.", port_alias.c_str(), table.description.c_str());
```

Comment:
would it be better to make this NOTICE level so we have visibility for pending ports?